man/systemd.network.xml

```diff
@@ -2570,7 +2570,8 @@ NFTSet=prefix:netdev:filter:eth_ipv4_prefix</programlisting>
           effect of the <option>Domains=</option> setting. If set to <option>route</option>, the domain name
           received from the DHCP server will be used for routing DNS queries only, but not for searching,
           similarly to the effect of the <option>Domains=</option> setting when the argument is prefixed with
-          <literal>~</literal>. Defaults to false.</para>
+          <literal>~</literal>. When unspecified, the value specified in the same setting in
+          <citerefentry><refentrytitle>networkd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>, will be used.</para>
```

Comment:
I'd like to keep `which defaults to <literal>no</literal>`. Otherwise, users need to jump to networkd.conf(5) and check the default explained there.

Of course, as commented, distribution may provide netowkrd.conf or its drop-in with non-default value, but that's not a problem here, as that's not the default value of networkd. The statement explains the default value, rather than the distro configuration, and does not contradict with the value explained in networkd.conf(5).

In short, please re-add the statement about the default value.